maths/factorial_python.py

```diff
@@ -1,19 +1,19 @@
-"""Python program to find the factorial of a number provided by the user."""
+# Python program to find the factorial of a number provided by the user.
 
 # change the value for a different result
-NUM = 10
+num = 7
 
 # uncomment to take input from the user
-# num = int(input("Enter a number: "))
+#num = int(input("Enter a number: "))
 
-FACTORIAL = 1
+factorial = 1
 
 # check if the number is negative, positive or zero
-if NUM < 0:
-    print("Sorry, factorial does not exist for negative numbers")
-elif NUM == 0:
-    print("The factorial of 0 is 1")
+if num < 0:
+   print("Sorry, factorial does not exist for negative numbers")
```

Comment:
Please revert this indentation change.  Four spaces are the standard for Python indentation.  See PEP8.